spec/models/candidate_interface/volunteering_role_form_spec.rb

```diff
@@ -6,7 +6,7 @@
       role: 'School Experience Intern',
       organisation: Faker::Educator.secondary_school,
       details: Faker::Lorem.paragraph_by_chars(number: 300),
-      working_with_children: [true, true, true, false].sample,
+      working_with_children: true,
```

Comment:
The intention of this is to test that it works with both false and true, but predominantly true; do we need to remove it?

Reply:
Good point will have a think.